product-tours/how-it-works/docs/Neptune-API-Tour.ipynb

```diff
@@ -121,12 +151,15 @@
    "source": [
     "import neptune.new as neptune\n",
     "\n",
-    "run = neptune.init(project=\"common/quickstarts\", api_token=\"ANONYMOUS\")"
+    "run = neptune.init(project=\"common/colab-test-run\", api_token=\"ANONYMOUS\")"
```

Comment:
Why changing project?

Reply:
Dashboard later in the notebook shows and links to the `colab-test-run` project
<img width="453" alt="image" src="https://user-images.githubusercontent.com/41324509/167266248-c4226074-32f4-4080-93ca-b3e1f1a8d8d7.png">